package.json

```diff
@@ -35,24 +35,27 @@
     "test": "istanbul cover _mocha -- --exit && npm run lint",
     "start": "electron .",
     "start-dev": "cross-env NODE_ENV=dev npm start",
-    "clean": "rm -rf builds config.json tasks.json coverage dist",
+    "clean": "rm -rf config.json tasks.json coverage dist",
     "lint": "eslint *.js app test",
-    "dist": "npm run clean && electron-builder -wlm"
+    "dist": "electron-builder",
+    "dist-all": "npm run clean && electron-builder --x64 --ia32"
```

Comment:
A few concerns:
* What does `electron-builder` (without arguments) do?
* Why not adding npm run clean to dist?
* Don't we need the -wlm option in `dist-all`?

Reply:
1. That `dist` script is there to build for all platform specified in `package.json` in the architecture currently running the script.
2. The `dist` script is thought to be more "manual" so to speak, it is meant to be used passing arguments so we don't have to make a script for each platform and/or target. Adding `npm run clean` would remove previous builds even if you are building to a platform not already built.
3. Not at all, it would package for all platforms and targets configured in `package.json`. Also this way you can specify to build to just one platform and/or target in both currently supported architectures, although it's discouraged because this script does remove all previous builds.

If you don't agree with the rationale, feel free to ask more questions or ask for specific fixes.

Reply:
1. Ok, I don't understand why, by default, `dist` only generates current OS build, while `dist-all` generates all by default without other arguments

2. I agree with passing arguments, but I'm not sure how those arguments are supposed to be passed, could you try and document this in the readme?

Reply:
I have tested it and I must have done something wrong, so dist-all has the same issue, my bad. So we do need `-wlm` in `dist-all`.
Passing arguments works this way, for example, `npm run dist -- --ia32 -w` launches the script and passes the arguments after `--`, so it builds the Windows targets in 32-bit, regardless of the OS you are running. I will document it.